### PR TITLE
docs: first pass at working with networks page

### PR DESCRIPTION
**Pull request by richardjgowers**
fixes issue #803

Checklist
* [ ] Added a ``news`` entry

## Developers certificate of origin
- [ ] I certify that this contribution is covered by the MIT License [here](https://github.com/OpenFreeEnergy/openfe/blob/main/LICENSE) and the **Developer Certificate of Origin** at <https://developercertificate.org/>.

### CODE REVIEW

File: docs/guide/results/working_with_networks.rst [hunk 1]
```diff
@@ -3,8 +3,56 @@
 Working with networks of results
 ================================
 
-issue #803
+After running a **network** of free energy calculations,
+we often want to consider analysing the corresponding network of results.
```
Comment (hannahbaumann) [inline docs/guide/results/working_with_networks.rst:7]:
```suggestion
we often want to analyse the corresponding network of results.
```
I think "consider" may not be necessary?

File: docs/guide/results/working_with_networks.rst [hunk 2]
```diff
@@ -3,8 +3,56 @@
...
 
+.. _userguide_MLE:
 
-The :ref:`openfe gather <cli_gather>` command offers a way to collate information across many different individual
+Converting relative results to absolute estimates
+-------------------------------------------------
+
+When a network of relative free energies have been calculated,
```
Comment (hannahbaumann) [inline docs/guide/results/working_with_networks.rst:14]:
```suggestion
When a network of relative free energies has been calculated,
```

File: docs/guide/results/working_with_networks.rst [hunk 3]
```diff
@@ -3,8 +3,56 @@
...
+a commonly performed task is to transform these pairwise estimations of **relative** free energies
```
Comment (hannahbaumann) [inline docs/guide/results/working_with_networks.rst:15]: Technically, we never obtain actual "free energies" but always "free energy differences" (so we don't get G but always Delta G or Delta Delta G). I know that always sounds very wordy so I'm not sure if it needs to be changed here or if we can leave it as is...

File: docs/guide/results/working_with_networks.rst [hunk 4]
```diff
@@ -3,8 +3,56 @@
...
+into a **absolute** free energies.
+This is done using a maximum likelihood estimator (MLE) [1]_,
+as implemented in the `cinnabar`_ package.
+This approach uses the matrix of relative pairwise measurements and their uncertainties,
+to estimate the overall ranking of
```
Comment (hannahbaumann) [inline docs/guide/results/working_with_networks.rst:20]: ranking of ...? (ligands? results?)

File: docs/guide/results/working_with_networks.rst [hunk 5]
```diff
@@ -3,8 +3,56 @@
...
+To use this approach the network of pairwise measurements needs to be fully connected,
+i.e. there should be a way to trace a path along pairwise measurements between any two nodes (ligands) on the network.
+
+.. note::
+   The results of a MLE estimation will have a **mean** of 0.0,
```
Comment (hannahbaumann) [inline docs/guide/results/working_with_networks.rst:25]: Do we have a code snippet somewhere (maybe in a cookbook?) that shows how one can shift the data?

File: docs/guide/results/working_with_networks.rst [hunk 6]
```diff
@@ -3,8 +3,56 @@
...
+
+      \sum_i^N \Delta G^i_{exp} = \sum_i^N \Delta G^i_{pred}
+
+Gathering using the command line
+--------------------------------
+
+After running calculations using the :ref:`quickrun command <userguide_quickrun>`,
+the :ref:`openfe gather <cli_gather>` command offers a way to collate information across many different individual
 simulations and prepare a table of results.
+The tool offers a summary of the relative binding affinities,
```
Comment (hannahbaumann) [inline docs/guide/results/working_with_networks.rst:40]: Would it make sense to add the command line option here?, e.g.
```suggestion
The tool offers a summary of the relative binding affinities (`--report ddg`),
```
and then in the next line ``--report dg``

File: docs/guide/results/working_with_networks.rst [hunk 7]
```diff
@@ -3,8 +3,56 @@
...
+Using cinnabar directly
+-----------------------
+
+The `cinnabar`_ package can be used from within Python to manipulate networks of free energy estimates.
+A tutorial on using this is provided here :ref:`here <tutorials/plotting_with_cinnabar.nblink>`
+
+See also
+--------
+
+* For handling the results of a single calculation, consult :ref:`userguide_individual_results`
```
Comment (hannahbaumann) [inline docs/guide/results/working_with_networks.rst:52]:
```suggestion
For handling the results of a single calculation, please consult :ref:`userguide_individual_results`
```
Looks a little bit weird to have a single bullet point here...

File: docs/guide/results/working_with_networks.rst [hunk 8]
```diff
@@ -3,8 +3,56 @@
...
 
+.. _userguide_MLE:
 
-The :ref:`openfe gather <cli_gather>` command offers a way to collate information across many different individual
+Converting relative results to absolute estimates
+-------------------------------------------------
+
+When a network of relative free energies have been calculated,
+a commonly performed task is to transform these pairwise estimations of **relative** free energies
+into a **absolute** free energies.
```
Comment (hannahbaumann) [inline docs/guide/results/working_with_networks.rst:16]: Either
```suggestion
into **absolute** free energies.
```
or
```suggestion
into a **absolute** free energy.
```

**Review by hannahbaumann — CHANGES_REQUESTED** [review]

File: docs/guide/results/working_with_networks.rst (hunk 5 above)
Comment (richardjgowers) [reply docs/guide/results/working_with_networks.rst:25]: not really, I need to add in this shifting to the cinnabar docs (and our gather command probably...)

File: docs/guide/results/working_with_networks.rst [hunk 9]
```diff
@@ -3,8 +3,56 @@
...
+we often want to analyse the corresponding network of results.
 
+.. _userguide_MLE:
 
-The :ref:`openfe gather <cli_gather>` command offers a way to collate information across many different individual
+Converting relative results to absolute estimates
+-------------------------------------------------
+
+When a network of relative free energies has been calculated,
+a commonly performed task is to transform these pairwise estimations of **relative** free energies differences (:math:`\Delta \Delta G`)
```
Comment (hannahbaumann) [inline docs/guide/results/working_with_networks.rst:15]:
```suggestion
a commonly performed task is to transform these pairwise estimations of **relative** free energy differences (:math:`\Delta \Delta G`)
```

File: docs/guide/results/working_with_networks.rst [hunk 10]
```diff
@@ -3,8 +3,56 @@
...
+      \sum_i^N \Delta G^i_{exp} = \sum_i^N \Delta G^i_{pred}
+
+Gathering using the command line
+--------------------------------
+
+After running calculations using the :ref:`quickrun command <userguide_quickrun>`,
+the :ref:`openfe gather <cli_gather>` command offers a way to collate information across many different individual
 simulations and prepare a table of results.
+The tool offers a summary of the relative binding affinities (`--report ddg`),
+or their :ref:`corresponding MLE values <userguide_MLE>`.
```
Comment (hannahbaumann) [inline docs/guide/results/working_with_networks.rst:41]:
```suggestion
or their :ref:`corresponding MLE values <userguide_MLE>` (`--report dg`).
```

**Review by hannahbaumann — CHANGES_REQUESTED** [review]

File: docs/guide/results/working_with_networks.rst [hunk 11]
```diff
@@ -3,8 +3,56 @@
...
+
+      \sum_i^N \Delta G^i_{exp} = \sum_i^N \Delta G^i_{pred}
+
+Gathering using the command line
+--------------------------------
+
+After running calculations using the :ref:`quickrun command <userguide_quickrun>`,
+the :ref:`openfe gather <cli_gather>` command offers a way to collate information across many different individual
 simulations and prepare a table of results.
+The tool offers a summary of the relative binding affinities (`--report ddg`),
```
Comment (hannahbaumann) [inline docs/guide/results/working_with_networks.rst:40]:
```suggestion
The tool offers a summary of the relative binding affinities (``--report ddg``),
```

**Review by hannahbaumann — APPROVED** [review]

**Pull request merged**